types/marked-terminal/package.json

```diff
@@ -1,13 +1,24 @@
 {
     "private": true,
     "name": "@types/marked-terminal",
-    "version": "3.1.9999",
+    "version": "6.0.9999",
     "projects": [
         "https://github.com/mikaelbr/marked-terminal"
     ],
+    "typeScriptVersion": "5.3",
```

Comment:
Due to the impending import attribute for importing ESM chalk types into a CJS marked-terminal file

Reply:
The PR still has it commented out; is this still needed? Or do we need to typesVersions this?

Reply:
I think it’s debatable. If we want to switch to using import attributes as soon as they’re available, we probably shouldn’t publish the ts-ignore version with no minimum and then bump the minimum really high on a patch release. If we’re comfortable with ts-ignore on this forever, we can remove it. Or, we can publish it both ways with `typesVersions`.

I guess setting the minimum version doesn’t actually stop most people from installing and trying a @types package, so even if we keep it, shipping the ts-ignore and then switching to import attributes could be disruptive. I guess we should either wait for import attributes to be available before merging, or use `typesVersions` once it’s available. I’ll remove it for now and plan on `typesVersions` in the future.